pedro-monteiro/1.ii/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Curriculum Vitae</title>
+</head>
+<body>
+    <div align="center">
+    <h1>Curriculum Vitae</h1>
+    
+    <img width="200px" height="200px" src="https://scontent-mia1-2.xx.fbcdn.net/v/t1.0-9/15894588_1265545726862866_700870937123917697_n.jpg?oh=4248f3cd5c34822793f1512db03a1921&oe=59464A98">
```

Comment:
Should I remove height and width? Because the image is disproportionate. @edysegura